geometry_msgs/Pose[] poses
---
int8 status
